modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,585 @@
+= XDCR Conflict Logging
+:description: pass:q[XDCR detects and logs concurrent conflicts that occur in different clusters but on the same document version due to independent modifications by locally connected applications during Active-Active replication. The conflict logs are for your information only. The best practice in Active-Active systems is that application environments must be designed to avoid conflicts.]
+:page-toclevels: 3
+
+[abstract]
+{description}
+
+During XDCR Active-Active replication, a conflict can occur if different applications connected locally to 2 different clusters, independently modify the same version of the document at around the same time.
+These simultaneous modifications happen within the convergence window*, so the changes made in one document are not reflected in the history of the other.
+
+If conflict logging is enabled, XDCR logs these conflict events in a conflict log collection, specified by you, for your information.
+The conflict logging does not change conflict resolution.
+
+XDCR performs conflict resolution and replicates the documents whether conflicts are logged or not.
+Conflict logging is informational only, so that you can review the conflict log message documents and the copies of the documents that were in conflict, to help you design application systems that avoid conflicts.
```

Comment:
Change "conflict log message documents" to "conflict log record documents" since that's the terminology we are using (as we explain later on).